### DISCUSSION TITLE
Cross Validation & bootstrapping for RSF

### DISCUSSION THREAD

**Discussion by nkaur201** [sebp/scikit-survival#331] [Q&A] [answered]
Hi, 

How would you perform cross validation or bootstrapping during the model building process for the random survival model. 

thanks,
Nkaur

**Comment by sebp** [comment, marked as answer]
The models in sksurv are compatible with scikit-learn, so you can just use [KFold](https://scikit-learn.org/1.1/modules/generated/sklearn.model_selection.KFold.html#sklearn.model_selection.KFold) to perform cross-validation. AFAIC, bootstrapping is not implemented in scikit-learn though.
